### DISCUSSION TITLE
How to input region specific permeability in PK-Sim?

### DISCUSSION THREAD

**Discussion by Prajwala1921** [Open-Systems-Pharmacology/Forum#1230] [Questions & Problems]
Hello all,
 I hope everybody is doing well, can anyone help me regarding an issue in PK-Sim. In the absorption window, there is an intestinal permeability option. But can I include the specific permeability in different segments of the intestine? If yes, anyone please help me do that. 
Thanks in advance!

**Comment by fredsevmartins** [comment]
Hi @Prajwala1921  yes, you can.
Open one simulation ... go to PARAMETER>ADVANCED (in the bottom)> PERMABILITY> P ENDOTHELIAL >MUCOSA.
Note: the changes in a simulation will not propagate to your building blocks.
Best regards,  

![image](https://user-images.githubusercontent.com/45317633/176752215-69af9de7-f225-4f3c-97fe-27c4929d4ba0.png)

**Reply by Prajwala1921** [reply]
Hi @fredsevmartins, 
Thanks a lot. It helps.
